client/src/components/QuestionModal/index.jsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react'
+import styles from './QuestionModal.css';
+
+class QuestionModal extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      question: '',
+      nickname: '',
+      email: ''
+
+    };
+  }
+
+  render() {
+    return (
+      <div className={styles.questionModalForm}>
+      <form className={styles.modalform}>
+        <input
+          type="text"
+          name="question"
+          placeholder="Enter your question"
+        />
+        <input
+          type="text"
+          name="nickname"
+          placeholder="Example: jackson11!"
+        />
+        <input
+          type="email"
+          name="email"
+          placeholder="enter email"
+        />
+        <small>For authentication reasons, you will not be emailed</small>
+      </form>
+        <button type="button">Submit</button>
```

Comment:
im fuzzy on this, does the button need to be inside the form element?

Reply:
I think the button needs to be in form element - unless there was a submit feature on the button separately. Or an onClick event handler.

Reply:
Yes, the button needs to inside the form.  thank you.